### PR TITLE
chore(es/preset-env): Upgrade `browserslist-rs`

### PR DESCRIPTION
**Pull request by g-plane**
The new version has fixed some bugs and added support of loading configuration.

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -271,6 +273,7 @@ dependencies = [
  "serde-wasm-bindgen",
  "serde_json",
  "thiserror",
+ "ureq",
```
Comment (kdy1) [inline Cargo.lock:276]: Seems like a wrong dependency?
Maybe `[dev-dependencies]` can be better.

File: Cargo.lock [hunk 2]
```diff
@@ -2074,6 +2083,21 @@ version = "0.1.4"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "448296241d034b96c11173591deaa1302f2c17b56092106c1f92c1bc0183a8c9"
 
+[[package]]
+name = "ring"
```
Comment (kdy1) [inline Cargo.lock:2087]: CI failed because of this.

**Review by kdy1 — CHANGES_REQUESTED** [review]

File: Cargo.lock (hunk 1 above)
Comment (g-plane) [reply Cargo.lock:276]: It's used in `build.rs`, so it's under `[build-dependencies]`.

File: Cargo.lock (hunk 2 above)
Comment (g-plane) [reply Cargo.lock:2087]: Should I switch to another HTTP client library?

Comment (kdy1) [reply Cargo.lock:2087]: I think it's preferrable. I checked `cargo-deny`, but there was no option to exclude build-time dependencies.

**Review by kdy1 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**